beacon_node/http_api/src/test_utils.rs

```diff
@@ -42,7 +43,7 @@ pub struct InteractiveTester<E: EthSpec> {
     pub harness: BeaconChainHarness<EphemeralHarnessType<E>>,
     pub client: BeaconNodeHttpClient,
     pub network_rx: NetworkReceivers<E>,
-    _server_shutdown: oneshot::Sender<()>,
+    _test_runtime: TestRuntime,
```

Comment:
I needed a `TestRuntime` to access an executor. Since the `TestRuntime` already includes shutdown senders, I managed to get rid of some existing ones.